File: solved.ac/code/18406.js
const fs = require("fs")
const num = fs.readFileSync("/dev/stdin").toString().trim()

let left = 0,
  right = 0

for (let i = 0; i < num.length / 2; i++) {
  left += +num[i]
  right += +num[num.length - i - 1]
}
console.log(left === right ? "LUCKY" : "READY")

File: solved.ac/python/18406.py
n = input()

n1 = n[0:len(n) // 2]
n2 = n[len(n)// 2 :]

n1_sum = 0 

for i in n1:
  n1_sum += int(i)

n2_sum = 0

for i in n2:
  n2_sum += int(i)

if n1_sum == n2_sum :
  print('LUCKY')
else:
  print('READY')